guides/Cloud_development/ZEIT.md

```diff
@@ -0,0 +1,35 @@
+# ZEIT
+**[ZEIT](https://zeit.co/)** with their mission - "To Make Cloud Computing Accessible To Everyone", which help developers to deploy their web projects with minimal effort and maintenance.
+ 
+
+## What is ZEIT Now?
```

Comment:
What is serverless APIs? Free automatic SSL?
Reduce the technical jargon as much as you can.

Reply:
Edited by reducing technical jargon. Please let me know if there is more to ament.